Rakefile

```diff
@@ -0,0 +1,10 @@
+require 'rake/clean'
```

Comment:
For context, how did you generate these files?

Reply:
I created it manually, the `with_unbundled_env` part by trial and error.

In case you were interested specifically in `rake/clean`, this is standard Rake, and it provides the `clobber` task borrowing the term from `make`, see [Clobbering - Wikipedia](https://en.wikipedia.org/wiki/Clobbering)
https://en.wikipedia.org/wiki/Clobbering